src/components/NationalText/utils.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { Fragment } from 'react';
+import AggregationsJSON from 'assets/data/aggregations.json';
+import { last, sum, isFinite } from 'lodash';
+import moment from 'moment';
+import {
+  formatPercent,
+  formatDecimal,
+  getPercentChange,
+  formatEstimate,
+} from 'common/utils';
+
+const getTwoWeekPercentChange = (series: any[]): number | null => {
+  const lastTwoWeeks = series.slice(-14);
+  const firstVal = lastTwoWeeks[0];
+  const lastVal = last(lastTwoWeeks);
+  if (!firstVal || !lastVal) {
+    return null;
+  }
+  return getPercentChange(firstVal, lastVal);
+};
+
+const getTotalCasesCopy = (summedRawCases: number): string => {
+  const ONE_MILLION = 1000000;
+  const divided = summedRawCases / ONE_MILLION;
+  return `${formatDecimal(divided, 1)} million`;
+};
+
+export const getNationalText: React.FC = () => {
+  const usaAggregation = AggregationsJSON['00001'];
+  const {
+    rawDailyCases,
+    rawDailyDeaths,
+    smoothedDailyCases,
+    smoothedDailyDeaths,
+    dates,
+  } = usaAggregation;
+
+  const totalCases: number = sum(rawDailyCases);
+
+  const totalDeaths: number = sum(rawDailyDeaths);
+
+  const percentChangeSmoothedCases = getTwoWeekPercentChange(
+    smoothedDailyCases,
+  );
+  const percentChangeSmoothedDeaths = getTwoWeekPercentChange(
+    smoothedDailyDeaths,
+  );
+
+  const renderPercentChangeSentence =
+    isFinite(percentChangeSmoothedCases) &&
+    isFinite(percentChangeSmoothedDeaths);
+
+  const lastDate = last(dates);
+  const lastDateFormatted: string = moment(lastDate).format('MMMM Do, YYYY');
+
+  const getChangeDescriptorCopy = (percentChange: number): string => {
+    const changeByCopy = `by about ${formatPercent(Math.abs(percentChange))}`;
+    if (percentChange > 0.01) {
+      return `increased ${changeByCopy}`;
+    } else if (percentChange < 0) {
```

Comment:
nit: do we want `< 0.01` so that we don't get "decreased 0%" text?

Reply:
if the change were 0%, it would go into the else ("remained about the same"), no?

Reply:
Oops, sorry. I meant `< -0.01`.  Right now, if the percentChange is e.g. `-0.004` then it'll round to `0%` and it'll say "decreased 0%", right?